dotnet/src/SemanticKernel.Abstractions/Kernel.cs

```diff
@@ -32,6 +32,10 @@ public sealed class Kernel
     private CultureInfo _culture = CultureInfo.InvariantCulture;
     /// <summary>The collection of plugins, initialized via the constructor or lazily-initialized on first access via <see cref="Plugins"/>.</summary>
     private KernelPluginCollection? _plugins;
+    /// <summary>Contains collection of function filters.</summary>
+    private readonly IList<IFunctionFilter>? _functionFilters;
+    /// <summary>Contains collection of prompt filters.</summary>
+    private readonly IList<IPromptFilter>? _promptFilters;
```

Comment:
With how you've written the initialization code, aren't these always non-null arrays rather than nullable ilists?

Reply:
Makes sense, updated them to non-null arrays.